src/runtime/js/ts.ui/core/core-gui@tradeshift.com/spirits/modals/ts.ui.ModalSpirit.js

```diff
@@ -269,6 +253,24 @@ ts.ui.ModalSpirit = (function using(Client, transition, chained) {
 		// Privileged ..............................................................
 
 		/**
+		 * Called by the {DoorManPlugin} when the modal opens.
+		 */
+		$onopen: function() {
+			this.css
+				.shift(this.$fullscreen, 'ts-fullscreen')
```

Comment:
I guess not if it's only used in one file, seeing that all these constants are half and anti-pattern to begin with (at least if the code is not rigged up to throw exceptions on `undefined` values, which the UI components are, so we should of course continue using them).